### DISCUSSION TITLE
How to upgrade flux in place installed with eksctl

### DISCUSSION THREAD

**Discussion by davisford** [fluxcd/flux2#1261] [Q&A] [answered]
```
✗  flux check
► checking prerequisites
✗ flux 0.12.1 <0.12.2 (new version is available, please upgrade)
✔ kubectl 1.20.4-dirty >=1.18.0-0
✔ Kubernetes 1.19.8-eks-96780e >=1.16.0-0
► checking controllers
✔ helm-controller: deployment ready
► ghcr.io/fluxcd/helm-controller:v0.9.0
✔ kustomize-controller: deployment ready
► ghcr.io/fluxcd/kustomize-controller:v0.11.0
✔ notification-controller: deployment ready
► ghcr.io/fluxcd/notification-controller:v0.12.0
✔ source-controller: deployment ready
► ghcr.io/fluxcd/source-controller:v0.11.0
✔ all checks passed
```

I bootstrapped flux with `eksctl`.  It installs an older version.  Before I just go edit the yaml in `gotk-components.yaml` I'd like to know if there is a CLI way I can force it to upgrade.

I tried the [bootstrap upgrade](https://toolkit.fluxcd.io/guides/installation/#bootstrap-upgrade) process and that did not upgrade anytyhing.

**Comment by hiddeco** [comment, marked as answer]
`flux 0.12.1 <0.12.2 (new version is available, please upgrade)` is about the binary (`flux`), not about the state of your controllers which are up to date on a quick glance.

You can find the latest binary [here](https://github.com/fluxcd/flux2/releases/latest) (or update it using your package manager).
